### ISSUE TITLE
How to specify Kubernetes version?

### ISSUE THREAD

**Issue by bcraun** [Azure/terraform-azurerm-aks#12]
The module has the Kubernetes version (1.11.3) hardcoded in the module which is unsupported in westus2. 

```
Error: Error applying plan:

1 error(s) occurred:

* module.aks.module.kubernetes.azurerm_kubernetes_cluster.main: 1 error(s) occurred:

* azurerm_kubernetes_cluster.main: containerservice.ManagedClustersClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="InvalidParameter" Message="The value of parameter orchestratorProfile.OrchestratorVersion is invalid." Target="orchestratorProfile.OrchestratorVersion"
```
I would like to specify a version by providing a value like the following: 

```
module "aks" {
  source  = "Azure/aks/azurerm"
  version = "0.9.0"

  CLIENT_ID = "${var.CLIENT_ID}"
  CLIENT_SECRET = "${var.CLIENT_SECRET}"
  location = "${var.location}"
  prefix = "${var.prefix}"
  kubernetes_version = "${var.kubernetes_version}"
}
```

I can change the module code in ```main.tf``` and ```variables.tf```, but am wondering if there a way to override the hardcoded value or if the module code does need to be changed? If so, I'm happy to submit a PR to add this support. Thanks.

**Comment by bummba** [comment]
So fare nobody cared much about our Pull Requests. In the meantime we have provided updated versions within our forks.
https://registry.terraform.io/modules/adfinis-forks
As soon as Microsoft cares, we might remove the forks again, but so fare this is not foreseeable.

**Comment by gambtho** [comment]
hey @bcraun I was looking for this same thing...does this commit solve what you were looking for? https://github.com/Azure/terraform-azurerm-aks/commit/a695ad1b0b403b2d3295bf0f506a38d077aa0af9

**Comment by n01d** [comment]
I am sorry for not taking care of the PRs until now.  We will review everything in the coming days. Thanks for your contributions and feedback.

**Comment by n01d** [comment]
This has been solved through #7 which is now published with [v1.0](https://github.com/Azure/terraform-azurerm-aks/tree/v1.0), thanks to @hairmare.

**Comment by bcraun** [comment]
@gambtho yes, it sure does. thanks @hairmare, @n01d.
